packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1067,6 +1067,35 @@ oie.registration.form.submit = Sign Up
 oie.user.profile.lastname= Last name
 oie.user.profile.firstname= First name
 oie.user.profile.primary.email= Email
```

Comment:
Pre-existing translations here. Using the same `oie.user.profile.` prefixing for new additions